src/qibocal/protocols/utils.py

```diff
@@ -772,3 +773,21 @@ def extract_feature(
     )
     second_mask = z[first_mask] < max if feat == "min" else z[first_mask] > min
     return x[first_mask][second_mask], y[first_mask][second_mask]
+
+
+def guess_period(x, y):
+    """Return fft period estimation given a sinusoidal plot."""
+
+    fft = np.fft.rfft(y)
+    fft_freqs = np.fft.rfftfreq(len(y), d=(x[1] - x[0]))
+    mags = abs(fft)
+    mags[0] = 0
+    local_maxima, _ = find_peaks(mags)
+    if len(local_maxima) > 0:
+        dominant_freq = fft_freqs[np.argmax(mags)]
+        return 1 / dominant_freq
+    return None
+
+
+fallback_period = lambda period: period if period is not None else 4
+"""Function to estimate period if guess_period fails."""
```

Comment:
Pylint will prefer:
```suggestion
def fallback_period(period):
    """Function to estimate period if guess_period fails."""
    return period if period is not None else 4
```
and we're not paying per line, so we could make him happy :P